src/Services/CartService.php

```diff
@@ -688,16 +683,17 @@ public function refreshCart()
      */
     public function isPaymentPlanEligible()
     {
+        if ($this->hasAnyRecurringSubscriptionProducts()) {
```

Comment:
This was a small performance improvement.
getDueForOrder is a fairly expensive operation and easy to avoid here.